### PR TITLE
task-1 add index skeleton

### PR DESCRIPTION
**Pull request by ikarpekmen**
- Add index skeleton

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    <main></main>
+<title>Page Title</title>
+</head>
+<body>
+
+<h1>My First Heading</h1>
+<p>My first paragraph.</p>
+
+</body>
+</html>
```
Comment (Pekmen) [inline index.html:14]: Please make code indented with 4 spaces

File: index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <main></main>
```
Comment (Pekmen) [inline index.html]: `<main>` tag should be inside body instead of head

File: index.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+<title>Page Title</title>
+</head>
+<body>
+
+<h1>My First Heading</h1>
+<p>My first paragraph.</p>
```
Comment (Pekmen) [inline index.html]: first paragraph should be in uppercase

**Review by Pekmen — CHANGES_REQUESTED** [review]

File: index.html (hunk 3 above)
Comment (ikarpekmen) [reply index.html]: there was not requirement for it to be uppercase

Comment (Pekmen) [reply index.html]: you are right, resolving this thread

**Review by Pekmen — APPROVED** [review]

**Pull request merged**